market/connector/ldap-connector/meta.json

```diff
@@ -3,7 +3,7 @@
 	"name": "Active Directory",
 	"description": "Read and write Microsoft Active Directory entries easily using LDAP.",
 	"type": "connector",
-	"vendor": "Axon Ivy AG",
+	"vendor": "",
```

Comment:
you need to delete the line :=)

Reply:
and you need to rebase from master :)
because the tests in your branch are still old